### ISSUE TITLE
UI behaviour suggestion - might be more streamlined if each tool remembers its own unique opacity setting

### ISSUE THREAD

**Issue by df-0** [nvrmind/IssueTracker#24]
Right now the opacity setting is the same for all tools, but I'm finding that I'm switching between (say) the paint brush at full opacity and then a very low opacity recolor tool, so I have to change the opacity every time I change the tool just to keep it the same - it would be more useful (for me personally) if the opacity slider changed the opacity for only the current tool and remembered what it had been set to for each tool?

**Comment by daseyb** [comment]
Good point :) Tools already remember their size, so that should be an easy addition
